src/components/Navbar.js

```diff
@@ -45,7 +45,7 @@ function Navbar() {
   return (
     <>
       <IconContext.Provider value={{ color: '#000' }}>
-        <div className={show ? 'navbar' : 'navbar hide-navbar'}>
+        <div className={show ? 'navbar show-navbar' : 'navbar hide-navbar'}>
```

Comment:
We are adding a `show-navbar` class, which I created, in order to show the "visible" styles of the navbar. The `hide-navbar` styles will be the opposite of that and the nav will alternate between them.